### ISSUE TITLE
no-empty-headings: False positives for curly contents

### ISSUE THREAD

**Issue by Turbo87** [ember-template-lint/ember-template-lint#2139]
The new `no-empty-headings` rule that was introduced by https://github.com/ember-template-lint/ember-template-lint/pull/2126 is causing false positives for things like:

```hbs
<h1>{{@title}}</h1>
```

/cc @glnster @bmish
